warp.sh

```diff
@@ -135,8 +159,8 @@ else
 	# Register a new account if no refresh token is provided
 	wg_private_key="$(wg genkey)"
 	wg_public_key="$(printf %s "${wg_private_key}" | wg pubkey)"
-	reg="$(cfcurl --header 'Content-Type: application/json' --request "POST" --header 'CF-Access-Jwt-Assertion: '"${teams_ephemeral_token}" \
-		--data '{"key":"'"${wg_public_key}"'","install_id":"","fcm_token":"","model":"","serial_number":"","locale":"en_US"}' \
+	reg="$(cfcurl --header 'Content-Type: application/json' --request "POST" --header "CF-Access-Jwt-Assertion: ${teams_ephemeral_token}" \
+		--data '{"key":"'"${wg_public_key}"'","install_id":"","fcm_token":"","model":"","serial_number":"","name":"","locale":"en_US"}' \
```

Comment:
What is this for?

Reply:
it came in as part #17 and i figured this was a formatting fix that worked - just added the empty name...

think need to autogen the name somehow or provide the option as a param